docs/static/monitoring/monitoring-apis.asciidoc

```diff
@@ -685,6 +687,18 @@ Example response:
   }
 --------------------------------------------------
 
+[discrete]
+[[geoip-database-stats]]
+==== Geoip database stats
+
+You can monitor stats for the geoip databases used with the <<plugins-filters-geoip, Geoip filter plugin>>.
+
+[source,js]
+--------------------------------------------------
+curl -XGET 'localhost:9600/_node/stats/geoip?pretty'
```

Comment:
We have decided to change the metrics holder from `geoip` to `geoip_download_manager`. So the API is
curl -XGET 'localhost:9600/_node/stats/geoip_download_manager?pretty'

Reply:
the decision is here https://github.com/elastic/logstash/pull/13004#issuecomment-866853878